Bottom: 707a09faefc0ab9f4339d4d2d9de51c28d1ba7a3
Top:    235cf0c788d32f5742914ca0e61448bcd2cdf205
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-11 19:10:01 +0000

Remove unused HistorySize field from the settings structure


---

diff --git a/src/global.h b/src/global.h
index 3794448..19f4454 100644
--- a/src/global.h
+++ b/src/global.h
@@ -668,8 +668,7 @@ typedef struct			/* some resources... */
     /* connections is done */
     AutoPlace;			/* flag which says we should force placement of the
 				   windows on startup */
-  int HistorySize,		/* FIXME? Used in hid/xaw code only. */
-    init_done;
+  int init_done;
 }
 SettingType, *SettingTypePtr;
